ckanext/xloader/jobs.py

```diff
@@ -285,16 +292,46 @@ def get_url():
     logger.info('Express Load completed')
 
 
+def get_response(url, headers):
+    def get_url():
```

Comment:
I would get rid of this extra method (`get_url`), since now you already created the `get_response` method.